pyproject.toml

```diff
@@ -33,6 +33,7 @@ typing_extensions = "^3.7.4"
 opentelemetry-api = {version = "^0.13b0",optional = true}
 opentelemetry-sdk = {version = "^0.13b0",optional = true}
 python-dateutil = "^2.7.0"
+pydantic = {version = "^1.6.1", optional = true}
```

Comment:
no idea, @patrick91? I think it doesn't include pydantic by default as it's defined in "extras" below

Reply:
the idea is that we don't require pydantic by default, so that if you're not using you don't need to install (same for django, flask, etc). 

You could install either manually or by doing `pip install strawberry-graphql[pydantic]` (and this would install both strawberry and pydantic at once :)